test/box_test.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:math';
+
+import 'package:file/memory.dart';
+import 'package:sqflite_common_ffi/sqflite_ffi.dart';
+import 'package:test/test.dart';
+
+import 'package:matrix/src/database/sqflite_box.dart';
+
+void main() {
+  group('Box tests', () {
+    late BoxCollection collection;
+    const Set<String> boxNames = {'cats', 'dogs'};
+    const data = {'name': 'Fluffy', 'age': 2};
+    const data2 = {'name': 'Loki', 'age': 4};
+    setUp(() async {
+      final fileSystem = MemoryFileSystem();
```

Comment:
This is useless, afaik? Just open a database names `:memory:`

Reply:
Done